app/controllers/api/messaging.php

```diff
@@ -1730,6 +1914,10 @@
             throw new Exception(Exception::MESSAGE_ALREADY_SENT);
         }
 
+        if (!is_null($message->getAttribute('deliveryTime')) && is_null($deliveryTime) && $message->getAttribute('deliveryTime') < new \DateTime()) {
```

Comment:
Let's remove `is_null($deliveryTime)` here, even if it wasn't changed, the exception should be thrown if the deliveryTime has passed

Reply:
done this